rpc/item.c

```diff
@@ -78,6 +79,11 @@ M0_TL_DESCR_DEFINE(pending_item, "pending-item-list", M0_INTERNAL,
 		   M0_RPC_ITEM_MAGIC, M0_RPC_ITEM_PENDING_CACHE_HEAD_MAGIC);
 M0_TL_DEFINE(pending_item, M0_INTERNAL, struct m0_rpc_item);
 
+M0_TL_DESCR_DEFINE(xidl, "rpc session xid list", M0_INTERNAL,
+		   struct m0_rpc_item, ri_xid_link, ri_magic,
+		   M0_RPC_ITEM_MAGIC, M0_RPC_ITEM_CACHE_HEAD_MAGIC);
```

Comment:
Here, we are supposed to use "M0_RPC_ITEM_XID_LIST_HEAD_MAGIC".

Reply:
Right. Fixed.